### PR TITLE
Style: Added Badge component

### PR DESCRIPTION
**Pull request by kalyandindu**
Hi!
Have created badge component for Icons and Avatars. Please do review and post your comments/suggestions if any

image
.

### CODE REVIEW

File: CSS/Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  height: 1rem;
+}
+.badge-lg {
+  width: 1.25rem;
+  height: 1.25rem;
+}
+.badge-notify {
+  width: 1.5rem;
+  height: 1.5rem;
+  top: 2px;
```
Comment (RahulSingh9131) [inline CSS/Components/badge.css:40]: Use rem here too.

**Review by RahulSingh9131 — COMMENTED** [review]
checked your code , everything is good. just replace px with rem to maintain the consistency, as you have used rem everywhere.

**Review by therajatg — COMMENTED** [review]
Looks good to me!

File: CSS/Components/badge.css (hunk 1 above)
Comment (kalyandindu) [reply CSS/Components/badge.css:40]: Sure, Thanks for the review

**Pull request merged**